build.rs

```diff
@@ -0,0 +1,52 @@
+fn main() {
+    #[cfg(target_os = "windows")]
+    download_winpcap_sdk()
+}
+
+#[cfg(target_os = "windows")]
+fn download_winpcap_sdk() {
```

Comment:
If I understand this correctly, this happens every time we build bandwhich, right? Even if the file already exists. Not terrible, but might make development a little slow? What do you think about using the `rerun-if-changed` flag on the resulting `Packet.lib` file? That way if it got deleted we'll redownload it, but otherwise skip this?

Reply:
Great idea, thanks. I thought that this flag was for C sources only, gotta check it out.